tests/test_flatland_malfunction.py

```diff
@@ -207,7 +208,7 @@ def test_malfunction_values_and_behavior():
 
     rail, rail_map, optionals = make_simple_rail2()
     action_dict: Dict[int, RailEnvActions] = {}
-    stochastic_data = MalfunctionParameters(malfunction_rate=1/0.001,  # Rate of malfunction occurence
+    stochastic_data = MalfunctionParameters(malfunction_rate=1 / 0.001,  # Rate of malfunction occurence
```

Comment:
```suggestion
    stochastic_data = MalfunctionParameters(malfunction_rate=1 / 0.001,  # Rate of malfunction occurrence
```
see:
https://www.merriam-webster.com/dictionary/occurence